src/js/libs/components/fetch.js

```diff
@@ -0,0 +1,49 @@
+function createNode(element) {
+    return document.createElement(element);
+}
+
+function append(parent, el) {
+  return parent.appendChild(el);
+}
+
+const tr1 = document.getElementById('loadBalancer1');
+
+export function getTableData() {
+    const response = fetch("https://westend.vedran.nodefactory.io/metrics")
+    .then(data => data.text())
+    .then(text => {
+        const activeNodes =  Number(/vedran_number_of_active_nodes\s*(\d*)\n/g.exec(text)[1]);
+        const penalizedNodes =  Number(/vedran_number_of_penalized_nodes\s*(\d*)\n/g.exec(text)[1]);
+        const totalNodes = activeNodes + penalizedNodes;
+        const loadBalancerName = "N/A"
+        const loadBalancerUrl = "https://westend.vedran.nodefactory.io/metrics"
+
+        //create <a>
+        let loadBalancerUrlA = createNode('a');
+        let loadBalancerUrlAText = document.createTextNode("link");
```

Comment:
```suggestion
        let loadBalancerUrlAText = document.createTextNode(loadBalancerUrl);
```
It doesn't have to be hyperlinked though, just plain text is fine